pages/admin/auth/login.tsx

```diff
@@ -111,9 +58,20 @@ const Login = ({ csrfToken, tenant, product }: InferGetServerSidePropsType<typeo
                 button: { 'data-testid': 'sso-login-button' },
               }}
             />
+            <div className='flex justify-center pt-4'>
+              <a
```

Comment:
From an accessibility point of view, this should be a button as we are not navigating away from the site instead just switching the view.